keyboards/niu_mini/keymaps/tobias/rules.mk

```diff
@@ -0,0 +1,6 @@
+ifndef QUANTUM_DIR
```

Comment:
This block should be remove, since it's not used.

Reply:
Removed the code.